packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx

```diff
@@ -123,9 +123,7 @@ const CurrentPregnancy: React.FC<PatientChartProps> = ({ patientUuid }) => {
       patientUuid,
       labourAndDeliveryEncounterType,
     );
-    const latestANCPTracker = getObsFromEncounter(currentPregnancyANCEncounter, pTrackerIdConcept);
-    const latestLnDPTracker = getObsFromEncounter(currentPregnancyLabourAndDeliveryEncounter, pTrackerIdConcept);
-    if (latestANCPTracker === latestLnDPTracker) {
```

Comment:
don't we need this logic anymore?

Reply:
No. This is what I explained in scrum yesterday. This logic assumed that all deliveries originated from ANC but there are cases of L & D without a corresponding ANC.